client/src/main/scala/org/http4s/client/oauth1/ProtocolParameter.scala

```diff
@@ -44,7 +45,12 @@ object ProtocolParameter {
 
   case class Custom(headerName: String, headerValue: String) extends ProtocolParameter
 
-  case class SignatureMethod(override val headerValue: String = "HMAC-SHA1")
+  object SignatureMethod {
+    val SHA1: SignatureMethod = SignatureMethod()
+    val SHA256: SignatureMethod = SignatureMethod(headerValue = "HMAC-SHA256", algorithm = "HmacSHA256")
```

Comment:
I'd prefix these constants with `Hmac`, because `RsaSha1` is another common option.